TYPO3.Flow/Classes/TYPO3/Flow/Resource/Target/FileSystemSymlinkTarget.php

```diff
@@ -45,27 +46,26 @@ public function publishCollection(CollectionInterface $collection)
      *
      * @param resource $sourceStream Stream of the source to publish
      * @param string $relativeTargetPathAndFilename relative path and filename in the target directory
-     * @throws Exception
-     * @throws \TYPO3\Flow\Utility\Exception
+     * @throws TargetException
```

Comment:
Does this really no longer throw an `Utility\Exception`? I think `createDirectoryRecursively` still could throw that